projects/ui-components/lf-metadata/lf-field-template-container/lf-field-template-container.component.ts

```diff
@@ -33,6 +34,25 @@ import { isDynamicField } from '../field-components/utils/metadata-utils';
 import { AppLocalizationService, FieldType } from '@laserfiche/lf-ui-components/shared';
 import { CoreUtils } from '@laserfiche/lf-js-utils';
 
+/**
+ * @internal
+ */
+ export enum DropDownLabelState {
```

Comment:
I'm not sure the label needs to change.. I think maybe it should still say no template assinged because there is not template assigned.

Reply:
the label doesn't need to change, but we need to the DropDownLabelState.HasError for the mat-options to shows that an error has occured